cmd/credential.go

```diff
@@ -29,7 +29,7 @@ const (
 	accessKeyMinLen = 5
 
 	// Maximum length for Minio access key.
-	accessKeyMaxLen = 20
+	accessKeyMaxLenMinio = 20
```

Comment:
Even this case `Minio` suffix is not conveying anything without reading the place it used.  IMO `accessKeyMaxLen` itself is sufficient.  If it looks generic `accessKeyGenMaxLen` would work.